### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DeniseSl22**
@marvinm2 added some documentation on error fixed today (docker was running, but not connecting to one of the ports)

### CODE REVIEW

**Comment by DeniseSl22** [comment]
@egonw I do not have push access to this repo

**Comment by egonw** [comment]
> I do not have push access to this repo

Neither do I, see the other PR :)

**Review by marvinm2 — APPROVED** [review]
Good additions. I will update these further after merge

**Pull request merged**
